index.html

```diff
@@ -18,27 +17,35 @@
   </head>
   <body>
     <div class="container">
-      <div class="introduction-header row">
-        <h1 class="header col s12"></h1>
-        <p class="slogan col s12"></p>
+      <div id="logo" class="introduction row">
```

Comment:
The class introduction is only used once. Does it need to be a class? You also have the id logo on this div, can that be used instead?